internal/trie/node/decode_test.go

```diff
@@ -98,66 +102,6 @@ func Test_Decode(t *testing.T) {
 				Dirty:    true,
 			},
 		},
-		"branch with two inlined children": {
```

Comment:
it is not possible to have to inline children in the branch node?

Reply:
Oh yes, it's just that test case is no longer needed since it's covered by a new test case in another test.